doc/leak_tracking/MOTIVATION.md

```diff
@@ -0,0 +1,81 @@
+
+Coming soon! See https://github.com/flutter/devtools/issues/3951.
+
+The text below is under construction.
+
+# Motivation for leak tracking
+
+This page describes motivation behind memory leak tracking.
+Read more about leak tracking in [overview](OVERVIEW.md).
+
+## Why do disposables exist?
+
+In most cases object lifecycle is managed well by a garbage collector.
```

Comment:
In most cases, object lifecycles are managed well by the Dart garbage collector.